store/tikv/2pc.go

```diff
@@ -431,6 +431,9 @@ func (c *twoPhaseCommitter) extractKeyExistsErrFromHandle(key kv.Key, value []by
 		if err != nil {
 			return c.genKeyExistsError(name, key.String(), err)
 		}
+		if len([]rune(str)) > col.Length {
+			str = string([]rune(str)[:col.Length])
```

Comment:
`col.Length` maybe be `-1` when it's `col1` without `(2)` :smile:

Reply:
fixed, PTAL @lysu